day1/Vagrantfile

```diff
@@ -2,32 +2,138 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.define "zabbix_server" do |server|
-    server.vm.hostname = "server"
-    server.vm.box = "sbeliakou/centos-7.3-x86_64-minimal"
-    server.vm.network "forwarded_port", guest: 80, host: 8080
-    server.vm.network "private_network", ip: "192.168.56.10"
-    server.vm.provider "virtualbox" do |vb|
-      vb.name = "Zabbix Server"
-      vb.memory = "1024"
+
+ config.vm.define "zabbix" do |zb|
+    zb.vm.box = "sbeliakou/centos-7.3-x86_64-minimal"
+    zb.vm.network "private_network", ip: "111.111.11.11"
+    zb.vm.network "public_network", bridge: "eno1"
+    zb.vm.hostname = "zabbix"
+    zb.vm.provider "virtualbox" do |v|
+	v.memory = "2048"
+	v.name = "zabbix"
     end
 
-    server.vm.provision "shell", path: "scripts/server.sh"
-    server.vm.provision "shell", path: "scripts/agent.sh"
+    zb.vm.synced_folder ".", "/home/vagrant/vb-share", create: true,
+      owner: "vagrant", group: "vagrant", type: "virtualbox"
+
+    zb.vm.provision "shell", inline: <<-SHELL
+	chattr -i /etc/selinux/config;
+	sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config;
+	systemctl stop firewalld
+	systemctl disable firewalld
+	yum install zip unzip net-tools git vim tree -y
```

Comment:
Basic start of SHELL-part for most of my Vagrantfiles since first usage of Vagrant at all. 
It makes my hands free for other jobs to be done.